test/parallel/test-npm-install.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const common = require('../common');
+
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+const fs = require('fs');
+
+common.refreshTmpDir();
+
+const npmPath = path.join(
+  common.testDir,
+  '..',
+  'deps',
+  'npm',
+  'bin',
+  'npm-cli.js'
+);
+
+const args = [
+  npmPath,
+  'install'
+];
+
+const pkgContent = '{}';
+
+const pkgPath = path.join(common.tmpDir, 'package.json');
+
+fs.writeFile(pkgPath, pkgContent, testNpm);
```

Comment:
since this is in test/parallel I wanted to keep things async here. Perhaps I'm over doing it.